ios/authentication.md

```diff
@@ -785,3 +819,402 @@ Note : `AWSGoogleSignIn` is only needed for using Google Login in your app and `
 	```
 
 Now, your drop-in UI will show a Google sign in button which the users can use to sign in to your app. This uses the `federatedSignIn()` flow underneath it.
+
+## Using Hosted UI for Authentication
+
+### Using Amazon Cognito Hosted UI 
+
+Amazon Cognito provides a customizable user experience via the hosted UI. The hosted UI supports OAuth 2.0 and Federated Identities with Facebook, Amazon, Google, and SAML providers. To learn more about Amazon Cognito Hosted UI, please visit [Amazon Cognito Developer Guide](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-configuring-app-integration.html).
+
+#### Setup your Cognito App Client
+
+To start using hosted UI, you need to configure your identity providers and setup your App Client in the Amazon Cognito console. You can also check the [Cognito doc: Adding Social Identity Providers to a User Pool](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-social-idp.html).
```

Comment:
@rohandubal to confirm is this copy/paste of the JS docs?

Reply:
rohan confirmed it was copy pasted. resolving.